.github/actions/next-stats-action/src/prepare/repo-setup.js

```diff
@@ -91,13 +91,14 @@ module.exports = (actionInfo) => {
                 continue
               }
               const pkgData = require(pkgDataPath)
-              const { name } = pkgData
+              const { name, version } = pkgData
               pkgDatas.set(name, {
-                pkgDataPath,
+                packedPkgPath,
                 pkg,
-                pkgPath,
                 pkgData,
-                packedPkgPath,
+                pkgDataPath,
+                pkgPath,
+                version,
```

Comment:
The order here changed but seems fine

Reply:
I sorted it by mistake, doesn't hurt though.